pyrh/robinhood.py

```diff
@@ -48,6 +48,27 @@ def investment_profile(self):
         """Fetch investment_profile."""
         return self.get(endpoints.investment_profile())
 
+    def get_instruments(self, symbol, match=True, options=False):
+    """Query for instruments that match with the given ticker.
+
+    Args:
+        symbol (str): stock ticker
+        match (bool): True if want exact match, False for partial match
+
+    Returns:
+        (:obj: (list)): JSON contents from `instruments` endpoint - list
+        of instruments that match the ticker
+    """
+    ticker = stock.upper()
+    params = {"symbol": ticker} if match else {"query": ticker}
+    res = self.get(endpoints.instruments(options=options), params=params)
+    results = res.get("results", [])
+    while res.get("next"):
+        res = res.get("next")
+        results.extend(res.get("results", []))
+    return results
+
+    @deprecated
```

Comment:
I wouldn't worry too much about marking functions as deprecated for now, I pegged a release out that was working 2.0 version and will likely mark all of the changes in the next version as 3.0

Reply:
Got it. I'll add some unit tests for the options model after work today.